### PR TITLE
Feature/storybook

### PR DESCRIPTION
**Pull request by Kerraan**

### CODE REVIEW

**Review by nathaliebru — APPROVED** [review]
Nice work!! ✨

File: apps/team-awesome-web/.storybook/main.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const path = require("path");
+
+module.exports = {
+  core: { builder: "webpack5" },
+  stories: ["../stories/**/*.stories.mdx", "../stories/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: [
+    "@storybook/addon-essentials",
```
Comment (b3rgt0p) [inline apps/team-awesome-web/.storybook/main.js:7]: ABC

File: apps/team-awesome-web/.storybook/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "outDir": ""
+  },
+  "files": [
+    "../../../node_modules/@nrwl/react/typings/styled-jsx.d.ts",
+    "../../../node_modules/@nrwl/react/typings/cssmodule.d.ts",
+    "../../../node_modules/@nrwl/react/typings/image.d.ts"
+  ],
+  "exclude": ["../**/*.spec.ts", "../**/*.spec.js", "../**/*.spec.tsx", "../**/*.spec.jsx"],
+  "include": [
+    "../components/**/*.stories.ts",
```
Comment (b3rgt0p) [inline apps/team-awesome-web/.storybook/tsconfig.json:14]: Components is probably redundant in this case

File: apps/team-awesome-web/src/components/button/Button.styled.ts [hunk 3]
```diff
@@ -26,6 +26,14 @@ export const Button = styled.button<ButtonProps>`
       color: ${({ theme }) => theme.colorSecondary};
     `}
 
+  :hover {
+    background-color: ${({ theme }) => theme.backgroundSecondary};
+  }
+
+  :active {
+    transform: translate(4px, 2px);
```
Comment (b3rgt0p) [inline apps/team-awesome-web/src/components/button/Button.styled.ts:34]: Is this a 'pressing animation'? Nice!

File: apps/team-awesome-web/stories/Button.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+import { action } from "@storybook/addon-actions";
+import { Button } from "../src/components/button";
+
+export default {
+  title: "Button",
```
Comment (b3rgt0p) [inline apps/team-awesome-web/stories/Button.stories.tsx:7]: ABC

**Review by b3rgundy — APPROVED** [review]
_Yeah, lesgo!_ 🥸

File: apps/team-awesome-web/src/components/button/Button.styled.ts (hunk 3 above)
Comment (Kerraan) [reply apps/team-awesome-web/src/components/button/Button.styled.ts:34]: Yes!!! You can do really cool stuff with this

**Pull request merged**